okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -163,29 +200,90 @@ class HttpLoggingInterceptor
 
     @Throws(IOException::class)
     override fun intercept(chain: Interceptor.Chain): Response {
-      val level = this.level
+      var request = chain.request()
+
+      val level = this.requestLevel(request)
```

Comment:
This is two different features in one PR... configurable levels per-request, and streaming